index.ts

```diff
@@ -32,3 +32,22 @@ const mutableContext = asMutable((await import('x')).context);
 export function asMutable<T>(value: T): Mutable<T> {
 	return value;
 }
+
+/**
+Check whether an array is empty.
+
```

Comment:
Could you include a sentence on why it's useful. The user will wonder why they should use this over a `.length` check.